src/main/webapp/assets/indexStyle.css

```diff
@@ -0,0 +1,209 @@
+#main-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 10vh 5%;
+}
+
+/*
+ * Title + subtitle
+ */
+
+#title-slogan {
+  color: var(--dark-green);
+  font-size: 4rem;
+  margin: 0;
+}
+
+.subtitle {
+  font-size: 1.2rem;
+  margin: 2rem 0;
+}
+
+#slogan {
+  flex: 0 1 30rem;
+  margin: 3rem 3rem 0 3rem;
+}
+
+#slogan-container {
+  display: flex;
+  flex-wrap: no-wrap;
+}
+
+#slogan-emoji-container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+/*
+ * Colleges dropdown
+ */
+
+#colleges-select {
+  margin: 0;
+}
+
+select:invalid {
+  color: gray;
+}
+
+select:focus {
+  border: 0 solid white;
+  outline: none;
+}
+
+.selectdiv {
+  position: relative;
+}
+
+select::-ms-expand {
+  display: none;
+}
+
+.selectdiv::after {
+  color: #565656;
+  content: '🛆';
+  font: 20px "Consolas", monospace;
```

Comment:
Nit: For CSS, I think Linter asks there to be a newline after the comma!

Reply:
I checked the google style guide and there's an [example](https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties) without the default font types on a new line + the super-linter didn't flag it, could you link to where the linter asks for newlines in this case? :0

Reply:
No worries then! I remembered that whenever I had commas within the same line, the linter got angry but if it doesn't for this, that's okay!

Reply:
Ok, sounds good!